docs/import-resolution.md

```diff
@@ -41,25 +41,25 @@ Pyright uses the following mechanisms (in priority order) to determine which Pyt
 3. As a fallback, use the default Python environment (i.e. the one that is invoked when typing `python` in the shell).
 
 ### Editable installs
-
-If you want to use static analysis tools with an editable install, you should configure the editable install to use `.pth` files that contain file paths rather than executable lines (prefixed with `import`) that install import hooks. See your package manager’s documentation for details on how to do this. We have provided some basic information for common package managers below.
+If you want to use static analysis tools with an editable install, you should configure the editable install to use `.pth` files that contain file paths rather than executable lines (prefixed with `import`) that install import hooks. Only setuptools is known to use import hooks by default, other build backends generally default to path-based `.pth` files.
```

Comment:
Nit: The phrase "other build backends..." should be a separate sentence.

Also, I recommend removing "generally" because it's ambiguous in this context. Does it mean that they sometimes don't default to path-based files? Does it mean that a small subset of them don't? If we're confident that setuptools is the only one, then we should say that. If we're not confident of this, then we should probably not call out setuptools specifically.

Reply:
In my user support experience in uv, the problems I saw were always setuptools. That doesn't mean other build backends can't do this, but so far as a gotcha that needs to be specifically documented I've only seen it from setuptools.